docs/deployment.md

```diff
@@ -0,0 +1,248 @@
+# Group Work v2 deployment 
+
+Right now Group Work v2 is not working in a Dogwood Edx installation, it requires a `edx-solutions` fork of the 
+platform, as it extensively uses the `edx-solutions` api. Some of the administrative tasks are also currently
+performed using proprietary Apros interface. 
+
+In order to have a working Group Project you'll need to complete three steps:
+
+1. Set up environment configuration variables
+2. Author the XBlock in Studio
+3. Configure Group Project in Apros (3rd party LMS) 
+
+## Apros configuration
+
+See corresponding document in Apros (TBD)
+
+![High level deployment overview](images/uml/high-level-architecture.png)
+
+# Setup
+
+In order to use Group Project XBlock v2 in a course, XBlock must be installed into LMS environment and enabled in 
+course's advanced settings. Also, Group Project XBlock uses a number of instance-specific configuration variables, so 
+setting them is considered a part of setup process.
+
+## Installing XBlock into LMS
+
+If you're using edx-solutions/edx-platform fork, Group Project v2 XBlock is already installed. Otherwise, installing 
+Group Project XBlock v2 is done the very same way other XBlocks are installed:
+
+1. Open terminal (or shell) to the LMS instance.
+2. Switch to LMS virtualenv
+3. pip install -e git+https://github.com/open-craft/xblock-group-project-v2.git@ **version_hash** #egg=xblock-group-project-v2
+
+*version_hash* is the hash of the git commit. By the time of writing, latest stable version was `4322ca8092c5385d5602143077e388017fcb1249`,
+so the above command looked like this:
+
+    pip install -e git+https://github.com/open-craft/xblock-group-project-v2.git@4322ca8092c5385d5602143077e388017fcb1249#egg=xblock-group-project-v2
+    
+## Setting configuration variables
+
+There are two sources of configuration variables: Django settings and XBlock-specific settings available 
+through [SettingsService][settings-service]. Both types of settings can be set via LMS environment file `lms.env.json`
```

Comment:
Nit: some of the settings (like `EDX_API_KEY`) are set in `lms.auth.json`, not `lms.env.json`